### PR TITLE
8150: Upgrade gems, rails, and dev goodies

### PR DESCRIPTION
**Pull request by meltheadorable**

### CODE REVIEW

**Comment by emitche** [comment]
@meltheadorable I created a new ticket for this work, so that it can be included in the list of tickets included the next time we do a release. https://redmine.sassafras.coop/issues/9021

File: .eslintrc.yml [hunk 1]
```diff
@@ -0,0 +1,105 @@
+env:
+  es6: true
+parserOptions:
+  ecmaFeatures:
+    jsx: true
+plugins:
+  - react
```
Comment (DevneyHamilton) [inline .eslintrc.yml]: Does Madeline have react? or is this a leftover from NEMO?

File: .eslintrc.yml [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+extends:
+  - eslint:recommended
+  - plugin:react/all
+globals:
+  React: true
+  ELMO: true
```
Comment (DevneyHamilton) [inline .eslintrc.yml]: ELMO looks like a holdover from NEMO as well that we probably don't want

File: .eslintrc.yml (hunk 1 above)
Comment (meltheadorable) [reply .eslintrc.yml]: some leftovers, yeah, thanks

**Pull request merged**